tests/smoke.fmf

```diff
@@ -0,0 +1,12 @@
+summary: Smoke test
+description: Import project main module in python.
+contact: Jiri Konecny <jkonecny@redhat.com>
+duration: 10m
+framework: shell
+path: /
+test: python3 -c "import initial_setup"
+
+# just avoid using local initial_setup folder
+adjust:
+  path: /data
```

Comment:
Not sure what do you mean by ``harmless subdir``. The adjust condition will take a place just when we have RPM file pre-installed which is happening on Packit. In that case we are getting to other place than project root to avoid import of the local ``initial_setup`` folder but instead use the system one.

Reply:
I suppose I misunderstood. I thought you want to *always* run the test in a directory which does *not* have the local source package with its python module directory, so that the python import will happen from the system-installed RPM path. But apparently you actually do want to test the git checkout in the "other" non-rpm case (whatever that is)?

Reply:
It's local. I'll try to improve the comment there. Seems it's not much helpful.

In general, when TMT is started by Packit/Gating it will also provide built package to TMT and this is installed to the system. You should always test these instead of source. However, when you run it locally (``make test``) then there is no automation to provide you this package, also it would take too much time when you need to build RPM for each test run. Because of that I'm adding the ``adjust``.

``adjust`` implementation in this case will change the path to not test source but RPM in case when it's started from Packit. That happenend thanks to the [trigger code](https://tmt.readthedocs.io/en/latest/spec/context.html#trigger).